### ISSUE TITLE
Can't disable emote text getting replaced by emote in chat field

### ISSUE THREAD

**Issue by Ayal1** [FrankerFaceZ/FrankerFaceZ#1207]
**Describe the bug**
When I type in an emote (Twitch sub or global, not third party ones like FFZ, BTTV, 7TV) it's text is automatically replaced by the actual image of it meaning if I want to add a lot of them, my chat input lags like crazy whereas before it wouldn't.

**To Reproduce**
1. Type in a Twitch global emote (such as "LUL")
2. See as it's replaced by the LUL emote instead of the text remaining.

**Expected behavior**
I expect the text of the emote to remain or to have the option to have it remain, even after tab auto completing.

**Comment by Lordmau5** [comment]
This is actually a new feature from Twitch itself - they have implemented a so called WYSIWYG editor for the chat input (What You See Is What You Get)

This is also the reason why FFZ (and presumably other extensions) don't support it with their own added emotes yet. It's a bit trickier to implement than how it was before since, well, it was just text before.

**Comment by Lordmau5** [comment]
From what I know there is no way to get back to the normal editor anymore... It annoys me deeply that Twitch didn't add a toggle in the Chat Settings to allow the emotes to be rendered in-place so people with not-so-well performing PCs can turn it off.

**Comment by Zoooook** [comment]
Workaround: https://github.com/FrankerFaceZ/FrankerFaceZ/issues/1197

**Comment by AlizarinRed** [comment]
Yeah, having the same issue here, we really should have a default option to turn this off, considering so many people tend to attach an emote to like "YES" and when someone comments something in excitement, "YES WE SHOULD DO THAT" it makes the YES an emote that might run the message.

Maybe like a multi-setting option:
Disable WYSIWYG
 - On All of Twitch 
 - On Only 3rd Party Emotes
 - etc.
